memgpt/schemas/openai/chat_completion_request.py

```diff
@@ -8,13 +8,15 @@ class SystemMessage(BaseModel):
     role: str = "system"
     name: Optional[str] = None
 
+class MultiModalMessage(BaseModel):
```

Comment:
We probably need to expand this to have `text` and `image_url`:

![image](https://github.com/user-attachments/assets/6bbb0470-7d32-4022-b5b9-32e72cd1e5b1)